packages/ai-jsx/readme.md

```diff
@@ -1,82 +1 @@
-
-## Quickstart
-
-1. Follow the [Getting Started Guide](https://docs.ai-jsx.com/getting-started)
-1. Run through the [tutorial](https://docs.ai-jsx.com/category/tutorial)
-1. Clone our [Hello World template](https://github.com/fixie-ai/ai-jsx-template) to start hacking
-1. Check out the different examples in the [examples package](https://github.com/fixie-ai/ai-jsx/tree/main/packages/examples)
-1. If you're new to AI, read the [Guide for AI Newcomers](https://docs.ai-jsx.com/guides/brand-new)
-
-## Examples
-
-Here is a simple example using AI.JSX to generate an AI response to a prompt:
-
-```tsx
-import * as AI from 'ai-jsx';
-import { ChatCompletion, UserMessage } from 'ai-jsx/core/completion';
-
-const app = (
-  <ChatCompletion>
-    <UserMessage>Write a Shakespearean sonnet about AI models.</UserMessage>
-  </ChatCompletion>
-);
-const renderContext = AI.createRenderContext();
-const response = await renderContext.render(app);
-console.log(response);
-```
-
-Here's a more complex example that uses the built-in `<Inline>` component to progressively generate multiple fields in a JSON object:
-
-```tsx
-function CharacterGenerator() {
-  const inlineCompletion = (prompt: Node) => (
-    <Completion stop={['"']} temperature={1.0}>
-      {prompt}
-    </Completion>
-  );
-
-  return (
-    <Inline>
-      Generate a character profile for a fantasy role-playing game in JSON format.{'\n'}
-      {'{'}
-      {'\n  '}"name": "{inlineCompletion}",
-      {'\n  '}"class": "{inlineCompletion}",
-      {'\n  '}"race": "{inlineCompletion}",
-      {'\n  '}"alignment": "{inlineCompletion}",
-      {'\n  '}"weapons": "{inlineCompletion}",
-      {'\n  '}"spells": "{inlineCompletion}",
-      {'\n}'}
-    </Inline>
-  );
-}
-```
-
-For a full set of examples, see [the examples package](https://github.com/fixie-ai/ai-jsx/tree/main/packages/examples).
-
-## Features
-
-- Prompt engineering through modular, reusable components
-- The ability to easily switch between model providers and LLM configurations (e.g., temperature)
-- Built in support for Tools (ReAct pattern), Document Question and Answering, Chain of Thought, and more
-- Ability to directly interweave LLM calls with standard UI components, including the ability for the LLM to render the UI from a set of provided components
-- Built-in streaming support
-- First-class support for NextJS and Create React App (more coming soon)
-- Full support for LangChainJS
-
-## Contributing
-
-We welcome contributions! See [Contributing](packages/docs/docs/contributing/index.md) for how to get started.
+# See [the root-level README](../../readme.md).
```

Comment:
@zkoch this file had already drifted from the root level file I believe it was intended to mirror. Lmk if you feel strongly about having the contents fully copied between the two.

I tried a symlink but GH doesn't render it nicely online.